applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -38,14 +41,13 @@ LISTIFY(CONFIG_BT_ISO_MAX_CHAN, NET_BUF_POOL_ITERATE, (;))
 	BT_LE_CONN_PARAM(CONFIG_BLE_ACL_CONN_INTERVAL, CONFIG_BLE_ACL_CONN_INTERVAL,               \
 			 CONFIG_BLE_ACL_SLAVE_LATENCY, CONFIG_BLE_ACL_SUP_TIMEOUT)
 
-#define DEVICE_NAME_PEER CONFIG_BT_DEVICE_NAME
-#define DEVICE_NAME_PEER_LEN (sizeof(DEVICE_NAME_PEER) - 1)
-
-static struct bt_conn *default_conn;
-static struct bt_audio_stream audio_stream;
+static struct bt_conn *headset_conn[CONFIG_BT_MAX_CONN];
+static struct bt_audio_stream audio_streams[CONFIG_BT_AUDIO_UNICAST_CLIENT_ASE_SNK_COUNT +
```

Comment:
Having streams for both sinks and sources, is that for bi-dir? If so, shouldn't be necessary yet